### ISSUE TITLE
no-duplicate-string false positive on imports

### ISSUE THREAD

**Issue by maximelkin** [SonarSource/SonarTS#700]
**I want to report a bug.**

**Rule key:** no-duplicate-string

**Reproducer** 
```
 // False-positive: tslint: Define a constant instead of duplicating this literal 8 times.
import {a} from './very_looooooooooooooooooooooooooooooooooong_name';
import {b} from './very_looooooooooooooooooooooooooooooooooong_name';
import {c} from './very_looooooooooooooooooooooooooooooooooong_name';
```

**Expected behavior**
No warning

**Comment by vilchik-elena** [comment]
@maximelkin why don't you merge imports? See this rule https://palantir.github.io/tslint/rules/no-duplicate-imports/

**Comment by maximelkin** [comment]
This is part of codestyle on existing project
